deploy/common/k8sbased.sh

```diff
@@ -650,5 +650,17 @@ function launch_sensor {
       "${COMMON_DIR}/monitoring.sh"
     fi
 
+    # If deploying with chaos proxy enabled, patch sensor to add toxiproxy proxy deployment
+    if [[ "$CHAOS_PROXY" == "true" ]]; then
+        original_endpoint=$(kubectl -n stackrox exec -it deploy/sensor -c sensor -- env | grep "ROX_CENTRAL_ENDPOINT" | cut -d'=' -f2)
```

Comment:
```suggestion
        original_endpoint=$(kubectl -n stackrox exec deploy/sensor -c sensor -- env | grep "ROX_CENTRAL_ENDPOINT" | cut -d'=' -f2)
```

The `-it` can be skipped for sure. I think, we could also avoid using bash and the pipe. Will see if I can come up with a proposal

Reply:
This does the job:

```
kubectl -n stackrox get deploy/sensor -ojsonpath='{.spec.template.spec.containers[0].env[?(@.name=="ROX_CENTRAL_ENDPOINT")].value}'
```

Reply:
Awesome, thanks!